_assets/stylesheets/elements/elements.css.scss

```diff
@@ -1,3 +0,0 @@
-@import "images.css.scss";
```

Comment:
Renamed.  If there's no leading `_`, the vars from the parent aren't available.  Which makes total sense o_O